easybuild/easyconfigs/m/MultiQC/MultiQC-1.7-intel-2018b-Python-3.6.6.eb

```diff
@@ -95,7 +95,12 @@ exts_list = [
     }),
     ('multiqc', version, {
         'source_urls': ['https://pypi.python.org/packages/source/m/multiqc/'],
-        'checksums': ['02e6a7fac7cd9ed036dcc6c92b8f8bcacbd28983ba6be53afb35e08868bd2d68'],
+        'patches': ['%(namelower)s-%(version)s_allow-matplotlib-3-for-python-3.patch'],
```

Comment:
@SethosII You're using `%(namelower)s` here, but the patch file uses `MultiQC` in the filename (rather than `multiqc`)